frontend/src/index.pug

```diff
@@ -41,14 +43,19 @@ html
 
         #tabs-wrapper(v-if="isTabActive")
           #tab-resize(onmousedown="registerMouseMove()")
-          .tab-close(v-on:click="isTabActive=false")
+          .tab-close(v-on:click="isTabActive=false;")
```

Comment:
doesnt seems like a necessary change for this PR; other lines are not practicing this.